### PR TITLE
LPS-130858

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://issues.liferay.com/browse/LPS-130858

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
@carolmariaabb, after this pull, feel free to send your pulls straightforward to Brian. :wink: If you need any assistance or questions talk to me before send the pulls, but you already understood the review workflow and I don't have much more things to teach you. In a near future, maybe we can make your review Gabriel's pulls and vice-versa.

Congratulations, @carolmariaabb! I'm very happy for you! :smile:

**Comment by carolmariaabb** [comment]
> @carolmariaabb, after this pull, feel free to send your pulls straightforward to Brian. 😉 If you need any assistance or questions talk to me before send the pulls, but you already understood the review workflow and I don't have much more things to teach you. In a near future, maybe we can make your review Gabriel's pulls and vice-versa.
> 
> Congratulations, @carolmariaabb! I'm very happy for you! 😄

Thanks for all the help, @natocesarrego ! 😄

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -69,7 +70,43 @@ const getDefaultRows = (nestedFields) => {
...
+					<ClayIcon symbol="question-circle-full" />
+				</span>
+			}
+		>
+			<p>{tooltip}</p>
+
+			<img
+				alt={Liferay.Language.get('input-mask-format')}
+				height={POPOVER_IMAGE_HEIGHT}
+				src={`${themeDisplay.getPathThemeImages()}/forms/input_mask_format.png`}
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:103]: @natocesarrego I was inspired by this example from App Builder: [WidgetTip.es.js](https://github.com/liferay/liferay-portal/commit/f05ee5358e10c3b4cbe36b179fc2b013a3a19888), that's why I put the image in this folder. 

> Are there any other images available on portal?

Yes. For example, the `geolocation.png` image is in this folder.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 2]
```diff
@@ -69,7 +70,43 @@ const getDefaultRows = (nestedFields) => {
...
+				width={POPOVER_IMAGE_WIDTH}
+			/>
+		</ClayPopover>
+	);
+};
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js]: Thanks 🥳

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/FieldBase/ReactFieldBase.es.js [hunk 3]
```diff
@@ -179,4 +204,34 @@ describe('ReactFieldBase', () => {
...
+		expect(getByRole('img')).toHaveAttribute('height', '170');
+		expect(getByRole('img')).toHaveAttribute(
+			'src',
+			'http://localhost:8080/forms/input_mask_format.png'
+		);
+		expect(getByRole('img')).toHaveAttribute('width', '232');
+
+		expect(getByText('input-mask-format')).toBeInTheDocument();
+		expect(getByText('Tooltip Description')).toBeInTheDocument();
+	});
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/FieldBase/ReactFieldBase.es.js:236]: 🤣🤣

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**